python/mach_bootstrap.py

```diff
@@ -160,7 +160,22 @@ def _activate_virtualenv(topdir):
         open(marker_path, 'w').close()
 
 
+def _ensure_case_insensitive_if_windows():
+    # The folder is called 'python'. By deliberately checking for it with the wrong case, we determine if the file
```

Comment:
This could just be, "By deliberately checking for the wrong case, we determine ..."